account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -1,4 +1,15 @@
-import { custom, parseEther, publicActions } from "viem";
+import {
+  custom,
+  parseEther,
+  publicActions,
+  getContract,
+  keccak256,
+  toHex,
+  hashTypedData,
+  hashMessage,
+  zeroAddress,
+  type TypedDataDefinition,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'TypedDataDefinition' is defined but never used.
<!-- __reviewdog__:ChA4YzAyMTNlZTUwNzc2NzU5EgZyZGpzb24= -->